README.md

```diff
@@ -1021,7 +1021,7 @@ function redirect(usr: Admin | User) {
 
 // Method 2: custom type guard, does the same thing in older TS versions or where `in` isnt enough
 function isAdmin(usr: Admin | User): usr is Admin {
-  return (<Admin>usr).role !==undefined
+  return (<Admin>usr).role !== undefined
```

Comment:
It probably makes more sense to simply use an `as any` cast since this would also work in a JSX file. Indifferent to this part. Would probably not recommend implementing type guards this way.

Reply:
yeah I would actually just change it to the `as` assertion syntax since this is a cheatsheet on TS and JSX/React anyways.

Reply:
Went with the `as any` cast. `as Admin` might cause issues with certain lint configurations because if `user` is of type `Admin` `user.role !== undefined` always evaluates to `true`.


Also renamed `usr` to `user` for readability and because `usr` means something completely different in certain tech contexts (see unix filesystem).